### ISSUE TITLE
Display bug with underscores on profile_proficiency.html

### ISSUE THREAD

**Issue by elthran** [elthran/RPG-Game#302]
The prof names should be using spaces and capitalization here, like: "Minimum Damage"
Currently it looks like "minimum_damage". Just need to fix that display.

**Comment by klondikemarlen** [comment]
I wrote a function in build_code.py that converts these.
Also you can get the normal name directly from 
```
import proficiencies

index = find('minimum_damage', proficiencies.ALL_ATTRIBUTE_NAMES)
proficiencies.ALL_NAMES[index]
```

**Comment by klondikemarlen** [comment]
Fixed in af0a67e6c. Moved `name`, `display_name` to Class level. Since they are now hardcode no calculation needs to take place.
